src/UniversalRewardsDistributor.sol

```diff
@@ -177,4 +176,14 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
         delete pendingRoot;
         emit EventsLib.RootSet(newRoot, newIpfsHash);
     }
+
+    function _setOwner(address newOwner) internal {
+        emit EventsLib.OwnerSet(owner, newOwner);
```

Comment:
Why do we want to log the previous owner? Isn't it supposed to be stored by integrators via previous events or default (0)?